update carbon ads placement

sayzlim opened the pull request:
- set the ad as fixed for viewport larger than 1028px
- set the ad in the footer viewport smaller than 1028px (better mobile support)

You can [check out the preview of the ad on three screen sizes at here](https://dropover.cloud/7077a5).

taylorotwell commented:
I'm not sure we want to show the ad on the main landing page though? Just on the docs?

sayzlim commented:
I've moved it back to docs page only. I added a new breakpoint to make the ad flows better on the mobile viewport. [Check out the preview](https://cln.sh/k6v95J). Let me know what you think.
